src/utils/person/personUtils.ts

```diff
@@ -8,3 +8,5 @@ export function showName(navn: Navn) {
 export function formatFnr(fnr: string) {
     return `${fnr.substr(0, 6)} ${fnr.substr(6, 11)}`;
 }
+
+export const er66EllerEldre = (alder: number | null) => (alder ?? 66) >= 66;
```

Comment:
Hva er tanken om default til 66 her, må vi ta høyde for at vi ikke har alder (og da defaulter vi til å vise advarselen)?

Reply:
Alder er lagt inn som nullable i interfacet ja, så defaultet bare til at den vises dersom vi faktisk skulle få `null` fra backend